etl_framework/BaseEtlSetUp.py

```diff
@@ -190,7 +196,12 @@ def set_etl_job_cutoff_value(self, datetime_cutoff=None):
         else:
             sql_statement = self.ETL_JOB_SELECT_CUTOFF_STATEMENT%(self.etl_job_id, )
 
-            datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            try:
+                datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            except IndexError:
+                # The ETL job has not yet been created, so we create it here
+                sql_statement = self.ETL_JOB_CREATE_JOB_STATEMENT%(self.etl_job_id, self.etl_job_name, )
+                self.sql_database.run_statement(sql_statement)
```

Comment:
@ianlofs It's actually pretty pythonic to do exception handling as conditional logic.  I'm worried that `IndexError` might be caused by another issue though. Is there a more specific way of capturing that the row doesn't exist @hdahme ??